### DISCUSSION TITLE
Oracle linux 9

### DISCUSSION THREAD

**Discussion by trondweiseth** [Security-Onion-Solutions/securityonion#12127] [2.4]
Tried to install Security Onion on Oracle 9 as per your documentation says is actually recommended to use.
I have tried and there is no logs to what is wrong. The only output i get is the following:
{
Getting started...
We do not support the version of CentOS you are trying to use.
}

I followed the insteallation documentation.

**Comment by reyesj2** [comment]
Looks like you tried to install Security Onion 2.3, which is EOL soon https://blog.securityonion.net/2024/01/3-month-end-of-life-eol-reminder-for.html
You should be installing Security Onion 2.4. https://github.com/Security-Onion-Solutions/securityonion/tree/2.4/main
